cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,31 @@
 /// <reference types='cypress' />
 
-describe('Book Store app', () => {
-  before(() => {
+const user = {
+  username: 'Anatolii Tsimbaliuk',
+  password: 'Qwert12345!',
+};
 
-  });
 
-  it('', () => {
+const title = 'Speaking JavaScript';
+const author = 'Axel Rauschmayer';
+const publisher = "O'Reilly Media";
+
+describe('Book Store app', () => {
+  it('should provide an opportunity to login', () => {
+    cy.visit('/login');
+    cy.login(user.username, user.password, 'login');
+    cy.contains('label', user.username).should('exist');
+    cy.url().should('include', '/profile');
+  });
 
+  it.only('should provide the ability for searching books', () => {
```

Comment:
```suggestion
  it('should provide the ability for searching books', () => {
```
I can't run all of your tests with this .only command. On the reals project it could be critical error, because you switch off all you tests except this